samples/bluetooth/peripheral_ancs_client/README.rst

```diff
@@ -0,0 +1,284 @@
+.. _peripheral_ancs_client:
+
+Bluetooth: Peripheral ANCS client
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Peripheral ANCS client sample demonstrates how to use the :ref:`ancs_client_readme`.
+
+Overview
+********
+
+The ANCS client sample implements an Apple Notification Center Service client. This client receives iOS notifications and is therefore a Notification Consumer. It can be connected with a Notification Provider, typically an iPhone or some other Apple device, which functions as ANCS server.
+
+When the sample is connected to a Notification Provider, it receives and prints incoming notifications on the UART.
+
+Notifications can have positive and negative actions associated with them, depending on the app that is sending the notification. For example, a notification for an incoming call is usually associated with the positive action to answer the call and the negative action to reject it. After receiving a notification, the available actions are indicated by flags on UART. The sample can perform the positive / negative action as response to the notification.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf52dk_nrf52832
+
+The sample also requires a device running a ANCS Server to connect with (for example, an iPhone which runs iOS, or a Bluetooth Low Energy dongle and nRF Connect for Desktop).
+
+User interface
+**************
+
+LED 1:
+   * Blinks with a period of 2 seconds, duty cycle 50%, when the main loop is running.
+
+LED 2:
+   * On when connected.
+
+Button 1:
+   * Request iOS notification attributes (content) on UART.
+
+Button 2:
+   * Request iOS app attributes on UART.
+
+Button 3:
+   * Perform a positive action as response to the last received notification.
+
+Button 4:
+   * Perform a negative action as response to the last received notification.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_ancs_client`
```

Comment:
This does not really have an effect, but please have an empty line after the heading.

Reply:
Done